src/main/java/mate/academy/internetshop/dao/jdbc/JdbcOrderDao.java

```diff
@@ -0,0 +1,144 @@
+        try (Connection con = ConnectionUtil.getConnection()) {
+            String query = "SELECT order_id FROM orders WHERE user_id = ?";
+            PreparedStatement statement = con.prepareStatement(query);
+            statement.setLong(1, userId);
+            ResultSet resultSet = statement.executeQuery();
+            List<Order> orders = new ArrayList<>();
+            while (resultSet.next()) {
+                orders.add(getOrder(resultSet.getLong("order_id"), userId));
+            }
+            return orders;
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    @Override
+    public Order create(Order order) {
+        try (Connection con = ConnectionUtil.getConnection()) {
+            String query = "INSERT INTO orders (user_id) VALUE (?)";
+            PreparedStatement statement = con.prepareStatement(query,
+                    PreparedStatement.RETURN_GENERATED_KEYS);
+            statement.setLong(1, order.getUser().getId());
+            statement.executeUpdate();
+            ResultSet resultSet = statement.getGeneratedKeys();
+            if (resultSet.next()) {
+                order.setId(resultSet.getLong(1));
+            }
+            saveProducts(order, con);
+            return order;
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    @Override
+    public Optional<Order> get(Long id) {
+        try (Connection con = ConnectionUtil.getConnection()) {
+            String query = "SELECT user_id FROM orders WHERE order_id = ?";
+            PreparedStatement statement = con.prepareStatement(query);
+            statement.setLong(1, id);
+            ResultSet resultSet = statement.executeQuery();
+            if (resultSet.next()) {
+                return Optional.of(getOrder(id, resultSet.getLong("user_id")));
+            }
+            return Optional.empty();
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    @Override
+    public List<Order> getAll() {
+        try (Connection con = ConnectionUtil.getConnection()) {
+            String query = "SELECT * FROM orders";
+            Statement statement = con.createStatement();
+            List<Order> orders = new ArrayList<>();
+            if (statement.execute(query)) {
+                ResultSet resultSet = statement.getResultSet();
+                while (resultSet.next()) {
+                    orders.add(getOrder(resultSet.getLong("order_id"),
+                            resultSet.getLong("user_id")));
+                }
+            }
+            return orders;
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    @Override
+    public Order update(Order order) {
+        try (Connection con = ConnectionUtil.getConnection()) {
+            String query = "UPDATE orders SET user_id = ? WHERE order_id = ?";
```

Comment:
In our case update order means update it's products. I'm not sure that we will pass order from one user to another in our internetshop